locals.tf

```diff
@@ -1,13 +1,16 @@
 locals {
-  project_name        = var.project_name
-  infrastructure_name = var.infrastructure_name
-  environment         = var.environment
-  aws_region          = var.aws_region
-  aws_account_id      = data.aws_caller_identity.current.account_id
-  resource_prefix     = "${var.project_name}-${var.infrastructure_name}-${var.environment}"
+  project_name         = var.project_name
+  infrastructure_name  = var.infrastructure_name
+  environment          = var.environment
+  aws_region           = var.aws_region
+  aws_account_id       = data.aws_caller_identity.current.account_id
+  resource_prefix      = "${var.project_name}-${var.infrastructure_name}-${var.environment}"
+  resource_prefix_hash = format("%.6s", sha1(local.resource_prefix))
```

Comment:
Could we make this 8 characters.

Reply:
Here I was following the same length hash that we decided to use on the account-bootstrap module: https://github.com/dxw/terraform-dxw-dalmatian-account-bootstrap/blob/main/locals.tf#L3

Reply:
okay. tool te to change now then :)